src/lib/convertParams.ts

```diff
@@ -33,3 +36,25 @@ export const toSpringSortParams = (elasticSortParams: ListingSortParams) => {
       return `${toCamelCase(sortType)},${convertedSortOrder}`
   }
 }
+
+const reverseSortLeadOrder = (sortOrder: LeadSortOrderParams) => {
+  switch (sortOrder) {
+    case LeadSortOrderParams.ASC:
+      return LeadSortOrderParams.DESC
+    case LeadSortOrderParams.DESC:
+      return LeadSortOrderParams.ASC
+  }
+}
+
+export const toSpringSortLeadParams = (elasticSortParams: LeadSortParams) => {
+  const { sortOrder, sortType } = elasticSortParams
+  const convertedSortOrder = toCamelCase(sortOrder)
+
+  switch (sortType) {
+    // We use alias in elastic, field name is createdDate
+    case LeadSortTypeParams.NEWEST:
+      return `createdDate,${toCamelCase(reverseSortLeadOrder(sortOrder))}`
+    default:
+      return `${toCamelCase(sortType)},${convertedSortOrder}`
+  }
+}
```

Comment:
Do we need to support both spring and elastic search here? The reason we have it for listings is that archived listings are handled with spring API whereas dealer and user listing searches are done via elastic and we want the parity between the two. Maybe we can simplify this if we only need to support spring?

Reply:
I just went for using same names convention for filters where `NEWEST` being the one used for `createdDate` on listing so I just used same here for leads. And from that made all the changes just to make it work.

Reply:
Removed spring helper functions part for leads. Won't even be using `NEWEST` as on leads we will use 'Date' name for sort and not 'Created' like we have on listing.